### PR TITLE
OCPQE-11842: Add e2e for ibmcloud-ipi

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by liangxia** [comment]
/cc @jhou1 @JianLi-RH @dis016 @pruan-rht 
PR ready for review

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -2231,6 +2231,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-upi-xpn
+- as: e2e-ibmcloud-ipi-p1
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    env:
+      E2E_RUN_TAGS: '@ibmcloud-ipi and @network-ovnkubernetes and not @fips'
```
Comment (MayXuQQ) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:2239]: no @network-ovnkubernetes

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:2239]: ovnkubernetes is the default network in OCP 4.12 unless openshiftsdn is defined explicitly.

**Pull request merged**